### PR TITLE
updating area ownership within mono and coreclr

### PR DESCRIPTION
**Pull request by jeffschwMSFT**
per our discussion, here are the updated area ownership for mono and coreclr

the sub area owners likely need feedback/updates

cc  @agocke, @lewing, @steveisok, @vitek-karas, @tommcdon, @mangod9

### CODE REVIEW

**Review by steveisok — APPROVED** [review]

**Review by mangod9 — APPROVED** [review]

**Review by lewing — APPROVED** [review]

File: docs/area-owners.md [hunk 1]
```diff
@@ -12,25 +12,24 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
+| area-CrossGen/NGEN-coreclr                     | @steveisok           | @dotnet/crossgen-contrib                            |                                                                                                                                                                                                                                                                                           |
+| area-crossgen2-coreclr                         | @steveisok           | @dotnet/crossgen-contrib                            |                                                                                                                                                                                                                                                                                           |
```
Comment (lambdageek) [inline docs/area-owners.md:27]: @steveisok do we need to add Fan, Zoltan and Kate to https://github.com/orgs/dotnet/teams/crossgen-contrib ?

File: docs/area-owners.md [hunk 2]
```diff
@@ -42,26 +41,25 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
-| area-GC-mono                                   | @SamMonoRT           | @BrzVlad                                            |                                                                                                                                                                                                                                                                                           |
+| area-GC-mono                                   | @mangod9             | @mangod9                                            |                                                                                                                                                                                                                                                                                           |
```
Comment (lambdageek) [inline docs/area-owners.md:44]: Should we add a note that @BrzVlad is a good person to consult?

```suggestion
| area-GC-mono                                   | @mangod9             | @mangod9                                            |                                                                                                                                                                                                                                                                                           | @BrzVlad to consult
```

File: docs/area-owners.md [hunk 3]
```diff
@@ -12,25 +12,24 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
+| area-Diagnostics-mono                          | @tommcdon            | @tommcdon @mdh1418                                  |                                                                                                                                                                                                                                                                                           |
+| area-EnC-mono                                  | @steveisok           | @lambdageek                                         | Hot Reload on WebAssembly, Android, iOS, etc                                                                                                                                                                                                                                              |
```
Comment (lambdageek) [inline docs/area-owners.md:32]: This is still us and not @tommcdon and @mikelle-rogers ?
```suggestion
| area-EnC-mono                                  | @tommcdon            | @mikelle-rogers                                          | Hot Reload on WebAssembly, Android, iOS, etc . @lambdageek to consult                                                                                                                                                                                                                                             |
```

**Review by lambdageek — APPROVED** [review]
looks ok; question about EnC in mono, and a suggestion about the mono GC

**Review by jkotas — APPROVED** [review]

**Review by vitek-karas — APPROVED** [review]

File: docs/area-owners.md [hunk 4]
```diff
@@ -42,26 +41,25 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
 | area-Infrastructure-libraries                  | @ericstj             | @dotnet/area-infrastructure-libraries               | Covers:<ul><li>Packaging</li><li>Build and test infra for libraries in dotnet/runtime repo</li><li>VS integration</li></ul><br/>                                                                                                                                                          |
 | area-Infrastructure-mono                       | @steveisok           | @directhex                                          |                                                                                                                                                                                                                                                                                           |
```
Comment (ericstj) [inline docs/area-owners.md:52]: @jeffschwMSFT @lewing @steveisok  who should this change to?

Comment (jeffschwMSFT) [reply docs/area-owners.md:52]: I left Jo in that space as he has a lot of knowledge and still works in the repo.  I can remove him for now.  We discussed a need for some infrastructure triage to sort this out (which is pending)

**Comment by jeffschwMSFT** [comment]
I am going to merge for now, but we can make additional changes as necessary (eg. Jo)

File: docs/area-owners.md (hunk 4 above)
Comment (ericstj) [reply docs/area-owners.md:52]: We can keep Jo listed as a consultant but not as owner / responsible party.

**Pull request merged**